src/web/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -155,7 +154,7 @@ const ColumnLinks: React.FC<{
                 { [firstColumnLinks]: index === 0 },
                 { [pillarColumnLinks]: !!column.pillar },
                 {
-                    [hide]: !showColumnLinks,
+                    [hideStyles]: !showColumnLinks,
```

Comment:
We don't need to change this here but just to say we have the convention of using `cx` to conditionally combine classes